ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -41,7 +59,7 @@ prefix_use_builtin_bootstrap: false
 prefix_custom_bootstrap_script:
   local: "{{ playbook_dir }}/../../bootstrap-prefix.sh"
   remote: /tmp/bootstrap-prefix.sh
-prefix_singularity_command: "singularity exec -B {{ gentoo_prefix_path }}:{{ gentoo_prefix_path }}"
+prefix_singularity_command: "singularity exec --env LD_LIBRARY_PATH= -B {{ gentoo_prefix_path }}:{{ gentoo_prefix_path }}"
```

Comment:
@trz42 We should add a comment above this line to clarify why LD_LIBRARY_PATH is explicitly set to empty?

Reply:
Because otherwise the bootstrap script simply stops running. It's explained in the PR.

Reply:
Added a comment.